state/linklayerdevices_test.go

```diff
@@ -1669,7 +1669,7 @@ func (s *linkLayerDevicesStateSuite) TestSetContainerLinkLayerDevicesMissingBrid
 	})
 	c.Assert(err, jc.ErrorIsNil)
 	err = s.machine.SetContainerLinkLayerDevices(s.containerMachine)
-	c.Assert(err.Error(), gc.Equals, `unable to find host bridge for spaces ["dmz"] for container "0/lxd/0"`)
+	c.Assert(err.Error(), gc.Equals, `unable to find host bridge for spaces "dmz" for container "0/lxd/0"`)
```

Comment:
this error strikes me odd, is there a possibility that there is a list of spaces? otherwise perhaps the error should be "for space" sorry for commenting this here and not in the error itself.